website/src/content/docs/docs/handbook/breaking-change-policy.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Breaking Change Policy
+---
+
+# Breaking Change Policy
+
+import { Badge } from "@astrojs/starlight/components";
+
+:::note
+This is to be applicable after 1.0.0 release
+:::
+
+## Versioning Scheme
+
+TypeSpec Compiler and libraries follow [semver](https://semver.org/) for versioning. Which means a breaking change(according the following policy) will only be introduced in a major version release.
```

Comment:
Honestly I'd leave this out. Semver is, I believe, quite prescriptive about what constittutes a breaking change, and stating this invites lawyering about that definition. Consider maybe: we follow a major, minor, patch versioning scheme according to the following policy: